Smart Grid general improvements (#48)
genexuslabs/web-controls-library: src/components/grid-fs/readme.md

# gx-grid-fs

> Container to include repetitive element list.

Reviewer: ```suggestion
A container for including a repetitive elements list.
```

It provides 3 slots:

> - 'grid-content' slot: Include the grid content here.

Reviewer: ```suggestion
- `'grid-content'` slot: This slot holds the grid's content
```

> - 'grid-content-empty' slot: This slot will be shown, only if the grid is empty.

Reviewer: ```suggestion
- 'grid-content-empty' slot: This slot will be shown only when the grid is empty.
```

> - 'grid-loading-content': Include the loading indicator markup here.

Reviewer: ```suggestion
- 'grid-loading-content': This slot will be shown while the grid is being loaded. Include the loading indicator markup here.
```

> When grid is empty, a CSS Class name 'gx-grid-empty' is added to the host element.

Reviewer: ```suggestion
When the grid is empty, a CSS Class named 'gx-grid-empty' is added to the host element.
```

> When grid is loading, a CSS Class name 'gx-grid-loading' is added to the host element.

Reviewer: ```suggestion
When the grid is loading, a CSS Class named 'gx-grid-loading' is added to the host element.
```
